frontend/package.json

```diff
@@ -2,10 +2,20 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
+  "config": {
+    "input": "src/",
+    "intermediate": "tmp/",
+    "output": "dist/"
+  },
```

Comment:
Since we're not using these config values in the scripts, we can remove this block entirely.

Reply:
This unnecessary part is removed.